src/lib/protocols/skype.c

```diff
@@ -1,128 +0,0 @@
-/*
- * skype.c
- *
- * Copyright (C) 2017-22 - ntop.org
- *
- * nDPI is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * nDPI is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with nDPI.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
-#include "ndpi_protocol_ids.h"
-
-#define NDPI_CURRENT_PROTO NDPI_PROTOCOL_SKYPE_TEAMS
-
-#include "ndpi_api.h"
-
-static int is_port(u_int16_t a, u_int16_t b, u_int16_t c) {
-  return(((a == c) || (b == c)) ? 1 : 0);
-}
-
-static void ndpi_check_skype(struct ndpi_detection_module_struct *ndpi_struct, struct ndpi_flow_struct *flow) {
-  struct ndpi_packet_struct *packet = &ndpi_struct->packet;
-  u_int32_t payload_len = packet->payload_packet_len;
-
-  /* No need to do ntohl() with 0xFFFFFFFF */
-  if(packet->iph
-     && ((packet->iph->daddr == 0xFFFFFFFF /* 255.255.255.255 */)
-	 || ((ntohl(packet->iph->daddr) & 0xFFFFFF00) == 0xE0000000 /* multicast */)
-	 )) {
-    NDPI_EXCLUDE_PROTO(ndpi_struct, flow);
-    return;
-  }
-
-  if(flow->host_server_name[0] != '\0')
-    return;
-  
-  if(packet->udp != NULL) {    
-
-    if(flow->packet_counter < 5) {
-      u_int16_t sport = ntohs(packet->udp->source);
-      u_int16_t dport = ntohs(packet->udp->dest);
-
-      /* skype-to-skype */
```

Comment:
is skype-to-skype protocol still recognized without this code? Did you check that?

Reply:
If with "skype-to-skype" you mean an interaction between "Skype-app" or "Skype-from-a-broswer" to/from "Skype-app" or "Skype-from-a-broswer", yes, I checked that. Always standard STUN or TLS stuff